floodviz/thumbnail/thumbnail.js

```diff
@@ -0,0 +1,90 @@
+// Dependency Import
+var fs = require('fs');
+var jsdom = require('jsdom/lib/old-api.js');
+var svg2png = require('svg2png');
+// Data imports
+var data_hydro = require('../thumbnail/hydrograph_data.json');
+
+// Collect script arguments for external css
+var style_path = null;
+var args = process.argv.splice(process.execArgv.length + 2);
+if (args.length > 2) {
+	console.log('\nUsage: node thumbnail.js ' +
+		'\n\nOptional flag: -css path/to/css/file.css\n');
+	process.exit();
+} else {
+	if (args[0] === '-css') {
+		style_path = args[1];
+	} else {
+		console.log('\nUnrecognized argument: ' + args[0] + '\n');
+		process.exit();
+	}
+}
+
+// Headless Browser Start for DOM
+jsdom.env(
+
+	// create DOM hook
+	"<html><body><div id='hydrograph'></div>" +
+	"<div id='map'></divid>" +
+	"</body></html>",
+
+	// load local assets into window environment
+	[
+		'./floodviz/static/bower_components/d3/d3.js',
+		'./floodviz/static/bower_components/proj4/dist/proj4.js',
+		'./floodviz/thumbnail/hydro_thumbnail.js',
+		'./floodviz/thumbnail/map_thumbnail.js'
+	],
+
+	function (err, window) {
+		var hydro_figure = window.hydromodule(
+			{
+				'height': 300,
+				'width': 560,
+				'div_id': '#hydrograph',
+				'data': data_hydro,
+				"display_ids": ['05471200', '05476750', '05411850', '05454220',
+				 '05481950', '05416900', '05464500', '05487470']
+				// Refactor Later. I'm assuming this will change with references.json
+			}
+		);
+		convert(hydro_figure,window,'floodviz/static/css/hydrograph.css','floodviz/thumbnail/thumbnail_hydro.png');
+	}
+);
+
+// Wrapper around svg2png that injects custom css to inline svg before conversion
+function convert(figure, window, css_path, filename) {
+	figure.init(undefined);
```

Comment:
It's better style to put this after the variable declarations. You can write this as figure.init().